woodwork/tests/accessor/test_statistics.py

```diff
@@ -1523,7 +1523,7 @@ def test_datetime_get_recent_value_counts():
 
 def test_numeric_histogram():
     column = pd.Series(np.random.randn(1000))
-    column.append(pd.Series([np.nan, " ", "test"]))
+    column = pd.concat([column, pd.Series([np.nan])])
```

Comment:
Looks like maybe this was broken previously as we just threw away the result of the previous append call. Should we update this to include `" "` and `"test"` like we intended to do before, or does it no make sense to have non-numeric values included for this test?

Reply:
This is something I wondered too, I think I made a Slack post about this earlier. It is worth noting that the inclusion of those strings leads the test to fail, so we would have to alter the expected result if we wanted to keep the strings.

Reply:
Not real sure why the were there to begin with, so seems fine to leave them out now.

Reply:
sounds good!